src/deepsparse/evaluation/registry.py

```diff
@@ -33,7 +33,12 @@ class EvaluationRegistry(RegistryMixin):
 
     @classmethod
     def load_from_registry(cls, name: str) -> Callable[..., "Result"]:  # noqa: F821
-        return cls.get_value_from_registry(name=name)
```

Comment:
@bfineran let's discuss whether the `RegistryMixin` should be robust against hyphens and underscores so that we could do something like

```python
@Registry.register(name=["bat_man", "cool-cat"])
class A

assert registry.load_from_registry("bat-man")
assert registry.load_from_registry("cool_cat")
```